README.md

```diff
@@ -179,6 +179,10 @@ a suggestion is attempted to be highlighted either by the keyboard or mouse. It
 scope, access to an object called `suggestion` which is the current suggestion that is being
 interacted with. A non-selectable suggestion cannot be clicked on, hovered over, or interacted with
 via the keyboard.
+ - `inputHint({query}) {Promise}`: An expression that hints at a suggestion by resolving a
```

Comment:
I think for the app-maker this could potentially be *much* simpler because some number of the time, the hint would just be the first result in the suggestions list if they set up their data correctly.